src/sbuf/SBuf.h

```diff
@@ -339,7 +339,7 @@ class SBuf
     SBuf consume(size_type n = npos);
 
     /// gets global statistic information
-    static const SBufStats& GetStats();
+    static const SBufStats & GetStats() { return Stats(); }
```

Comment:
We should not have two class methods named GetStats() and Stats(). The names should tell the two methods apart. This was discussed four days ago in another very similar PR: https://github.com/squid-cache/squid/pull/1749#discussion_r1534921622. Let's reuse the same approach here:

* Use WriteableStats() name for the new method.
* Use GetStats() to preserve old method callers and use a simpler name for the safer method. This method should have been named Stats(), but we do not need to improve that in this PR.

```suggestion
    static const auto &GetStats() { return WriteableStats(); }
```

If you prefer to also rename GetStats() to Stats() in this PR, I will support that change as well, but I do not insist on it. Your call.

Reply:
IIRC the `GetStats()` exists because you insisted on the single caller (mgr dump function) being provided a `const`. Statistics objects are generally always writable in Squid so they can be updated by the caller/user code, so calling it `WritableStats` is redundant.

Reply:
> Alex: We should not have two class methods named GetStats() and Stats(). The names should tell the two methods apart. This was discussed four days ago in another very similar PR: [#1749 (comment)](https://github.com/squid-cache/squid/pull/1749#discussion_r1534921622). Let's reuse the same approach here

> Amos: IIRC the `GetStats()` only exists because you insisted on the single caller (mgr dump function) being provided a `const`. Statistics objects are generally always writable in Squid so they can be updated by the caller/user code, so calling it `WritableStats` is redundant.

The response quoted above clearly does not address my concern about PR code/changes.

I do not see "redundancy" in "[Get]Stats() and WriteableStats()" naming scheme. If you reject the proposed solution (mimicking the one developed in #1749), please propose something better than that solution. Hopefully, both PRs (and future PRs!) can use the same overall approach that we can agree on.